src/proj.h

```diff
@@ -341,6 +341,9 @@ void proj_errno_restore (PJ *P, int err);
 
 char *proj_get_definition (PJ *P);
 
+/* deallocate anything obtained from PROJ.4 */
```

Comment:
Hum proj_definition_create() is a bit confusing for something that is essentially a getter